app/controllers/admin/orders_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Admin
+  class OrdersController < AdminController
+    def index
+      @orders = Order.all.includes([:employee]).order(:status)
+    end
+
+    def update
+      @order = Order.find(params[:id])
+      @order.status = 1
```

Comment:
try using helper method offered by enum here - something like
```suggestion
      @order.status_delivered!
```

Reply:
I have revised this method. Thank you for your advice. 

As for the commit descriptions - I promise to improve ;) I'll remind you at the next PR to see if anything else can be improved.